azure-pipelines/end-to-end-tests-dir/build-test-ports.ps1

```diff
@@ -2,4 +2,6 @@
 
 $CurrentTest = "Build Test Ports"
 
+Run-Vcpkg --overlay-ports="$PSScriptRoot/../e2e_ports/overlays" install vcpkg-empty-port
 Run-Vcpkg --overlay-ports="$PSScriptRoot/../e2e_ports" install vcpkg-find-acquire-program
+Throw-IfFailed

```

Comment:
@BillyONeal: Might want to fix this e2e test first for other platforms then windows. In the current state it will do nothing but with this PR it will fail on osx and linux since `vcpkg-find-acquire-program` only supports windows. So what was this test trying to do originally?

Reply:
Ouch, it's missing a ThrowIfFailed :(

Reply:
I can change `vcpkg-find-acquire-program` to [vcpkg-internal-e2e-test-port](https://github.com/microsoft/vcpkg-tool/tree/main/azure-pipelines/e2e_ports/vcpkg-internal-e2e-test-port) if that still covers the intention of the test. But I believe you explicitly wanted a install of  `vcpkg-find-acquire-program` somewhere

Reply:
I think you should add the missing ThrowIfFailed, and guard that test with `if ($IsWindows)`, and yell at the idiot to wrote the test to fix it for other platforms. Oh hey, we already did that second part 😅